src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -418,4 +423,80 @@ private MobileAdsDataProvider() {
         {"project43", "SyntheticTests/AdType/Async/Hop/ExtraMarker", "mobile", "\"test-marker\":\"42\""}
     };
   }
+
+  @DataProvider
+  public static Object[][] adsSlotSizeMercury() {
+    return new Object[][]{
+        {
+            new Page("adtest", "SyntheticTests/MobileLeaderboard"),
+            "",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.MOBILETOP_LB)
+                .put("slotSize", new Dimension(320, 100))
+                .put("lineItemId", 136987812)
+                .put("src", "mobile")
+                .build()
+        },
+        {
+            new Page("project43", "SyntheticTests/Slots/Size/320x50"),
+            "",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.MOBILETOP_LB)
+                .put("slotSize", new Dimension(320, 50))
+                .put("lineItemId", 257602332)
+                .put("src", "mobile")
+                .build()
+        },
+        {
+            new Page("project43", "SyntheticTests/Slots/Size/320x50"),
+            "",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.MOBILE_AD_IN_CONTENT)
+                .put("slotSize", new Dimension(320, 50))
+                .put("lineItemId", 257602332)
+                .put("src", "mobile")
+                .build()
+        },
+        {
+            new Page("project43", "SyntheticTests/Slots/Size/320x50"),
+            "",
+            ImmutableMap.<String, Object>builder()
+                .put("slotName", AdsContent.MOBILE_PREFOOTER)
+                .put("slotSize", new Dimension(320, 50))
+                .put("lineItemId", 257602332)
+                .put("src", "mobile")
+                .build()
+        },
+        {
+            new Page("project43", "SyntheticTests/Slots/Size/300x50"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SyntheticTests/Slots/Size/300x50" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)